ethcore/res/ethereum/classic.json

```diff
@@ -44,8 +44,8 @@
 		"eip1283Transition": "0xa03ae7",
 		"eip1344Transition": "0xa03ae7",
 		"eip1706Transition": "0xa03ae7",
-		"eip2028Transition": "0xa03ae7",
-		"eip2200AdvanceTransition": "0xa03ae7"
+		"eip1884Transition": "0xa03ae7",
+		"eip2028Transition": "0xa03ae7"
```

Comment:
What happened to 2200?

Reply:
```suggestion
		"eip2028Transition": "0xa03ae7"
		"eip1283ReenableTransition": "0xa03ae7",
```

Reply:
In Parity, 2200 is 1283 + 1706.

see https://github.com/OpenEthereum/open-ethereum/pull/11068#issuecomment-533104200

see https://github.com/OpenEthereum/open-ethereum/pull/11228#issuecomment-548730603

Reply:
your syntax is misplacing the commata

Reply:
Fixed the comma.

My reasoning was just to copy the same pattern as foundation.json, 	
```
"params": {
		"gasLimitBoundDivisor": "0x400",
		"registrar": "0xe3389675d0338462dC76C6f9A3e432550c36A142",
		"accountStartNonce": "0x0",
		"maximumExtraDataSize": "0x20",
		"minGasLimit": "0x1388",
		"networkID": "0x1",
		"forkBlock": "0x1d4c00",
		"forkCanonHash": "0x4985f5ca3d2afbec36529aa96f74de3cc10a2a4a6c44f2157a57d2c6059a11bb",
		"eip150Transition": "0x259518",
		"eip160Transition": "0x28d138",
		"eip161abcTransition": "0x28d138",
		"eip161dTransition": "0x28d138",
		"eip155Transition": "0x28d138",
		"maxCodeSize": "0x6000",
		"maxCodeSizeTransition": "0x28d138",
		"eip140Transition": "0x42ae50",
		"eip211Transition": "0x42ae50",
		"eip214Transition": "0x42ae50",
		"eip658Transition": "0x42ae50",
		"eip145Transition": "0x6f1580",
		"eip1014Transition": "0x6f1580",
		"eip1052Transition": "0x6f1580",
		"eip1283Transition": "0x0",
		"eip1283DisableTransition": "0x0",
		"eip1283ReenableTransition": "0x8a61c8",
		"eip1344Transition": "0x8a61c8",
		"eip1706Transition": "0x8a61c8",
		"eip1884Transition": "0x8a61c8",
		"eip2028Transition": "0x8a61c8"
	},
```

Reply:
according to the second link you have there, (https://github.com/OpenEthereum/open-ethereum/pull/11228#issuecomment-548730603)

> In Parity, 2200 is 1283 + 1706.

is incorrect. It's __`eip1283_reenable_transition`__ and `eip1706_transition`